### ISSUE TITLE
Issue building latest version on Mac (x86_64)

### ISSUE THREAD

**Issue by nicksrandall** [denoland/rusty_v8#819]
After upgrading to the latest version of `rusty_v8` (and `deno_core`), I'm no longer able to build locally due to an issue of `librusty_v8.a` file not being downloaded properly. I've tried deleting the file, cleaning my cargo cache and rebuilding but that doesn't seem to fix the issue. 

This very well could be local config issue but I've spent a few days trying to debug and I can't get it to work.

Here is a minimal repo that demonstrates the issue for me: https://github.com/nicksrandall/deno_build_issue

Here is the output from the issue I'm getting: 

```
error: failed to add native library /Users/{{USERNAME}}/code/deno_demo/target/debug/gn_out/obj/librusty_v8.a: file too small to be an archive

error: could not compile `rusty_v8` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

The output of the downloaded `librusty_v8.a` file is: 

```
Not Found%
```

**Comment by flaki** [comment]
Same/similar error on Linux:

```
...
   Compiling swc_macros_common v0.3.3
   Compiling swc_eq_ignore_macros v0.1.0
   Compiling auto_impl v0.4.1
   Compiling zeroize_derive v1.2.0
   Compiling is-macro v0.1.9
error: failed to add native library /tmp/cargo-installgec7d6/release/gn_out/obj/librusty_v8.a: file too small to be an archive

error: could not compile `rusty_v8` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `deno v1.15.3`, intermediate artifacts can be found at `/tmp/cargo-installgec7d6`

Caused by:
  build failed
```

**Comment by piscisaureus** [comment]
Might be related to https://github.com/denoland/rusty_v8/commit/5ee0a375f252e880d809d48dcd4390aee84b3b4a

**Comment by lucacasonato** [comment]
> Might be related to https://github.com/denoland/rusty_v8/commit/5ee0a375f252e880d809d48dcd4390aee84b3b4a

The version @flaki and @nicksrandall are using do not include that commit, so I don't think so.

**Comment by flaki** [comment]
```
flaki:/tmp/cargo-installgec7d6/release/gn_out/obj$ ll
total 12K
drwxrwxr-x 2 flaki flaki 4.0K Oct 31 08:33 ./
drwxrwxr-x 3 flaki flaki 4.0K Oct 31 08:33 ../
-rw-rw-r-- 1 flaki flaki    9 Oct 31 08:33 librusty_v8.a

flaki:/tmp/cargo-installgec7d6/release/gn_out/obj$ cat librusty_v8.a 
Not Found
```

It looks like the precompiled library download is 404-ing, if you can help me with how can I fish out the request URL would be happy to post it here if that helps debugging.

**Comment by nicksrandall** [comment]
I've tried to build on other platforms using "cargo cross" and they have also failed with similar errors.

**Comment by flaki** [comment]
I'm on Ubuntu 20.04.3 LTS (x86_64-linux-gnu) fwiw

**Comment by nicksrandall** [comment]
Upgrading `rusty_v8` 0.32 to `v8` 0.33 fixed the problem for me.

**Comment by lucacasonato** [comment]
> Upgrading `rusty_v8` 0.32 to `v8` 0.33 fixed the problem for me.

That does not solve the root problem.

**Comment by markuskobler** [comment]
It looks like the `librusty_v8.a` quietly fails to download. I have not been able to dig into this enough but could it have anything todo with dependencies resolving to `rusty_v8 v0.32.1 (*)` and it looks like there was only ever a `v0.32.0` [published release](https://github.com/denoland/rusty_v8/releases).

**Comment by lucacasonato** [comment]
This should be fixed now. We published a 0.32.1 to display a "renamed `rusty_v8` to `v8`" on crates.io, but forgot to actually upload binaries for it on GitHub.com. We have now yanked this release. Sorry for the issues!

**Comment by lucacasonato** [comment]
And we have now uploaded the binaries for 0.32.1, so the 0.32.1 release has been unyanked.
